src/index.ts

```diff
@@ -25,6 +26,7 @@ export default (
   }),
 ): Logger => {
   opts.redact = redact.addDefaultRedactPathStrings(opts.redact);
+  opts.redact = redact.addRemovePathsCensor(opts.redact);
```

Comment:
?

Reply:
Would a better name be `configureCensor`?

Reply:
I'm a bit sceptical of this chaining of two standalone redaction functions. Is it obvious how they both modify `opts.redact`, and what guarantees that they evolve in a compatible, non-conflicting manner?

At minimum we will want tests that cover combinations of both, especially if we ponder adding default redaction config.

Reply:
Maybe I should have an orchestration function to run them in the order of "add default redact paths" then "configure redact censor"?

Reply:
I've exported a `configureRedact` function that ensures the default redact paths are added before the censor is configured.